graphene_django_cud/mutations/batch_delete.py

```diff
@@ -104,12 +122,10 @@ def mutate(cls, root, info, ids):
             qs_to_delete = updated_qs
 
         # Find out which (global) ids are deleted, and which were not found.
-        deleted_ids = [
-            to_global_id(get_global_registry().get_type_for_model(Model).__name__, id)
-            for id in qs_to_delete.values_list("id", flat=True)
-        ]
+        deleted_ids = [cls.get_return_id(id) for id in qs_to_delete.values_list("id", flat=True)]
+
+        all_global_ids = [cls.get_return_id(id) for id in ids]
```

Comment:
could argue `set(map(cls.get_return_id, ids))` would read better. But I'm not looking to rock the boat too much!